### DISCUSSION TITLE
从动态预览进入视频后动态栏不会自动收起

### DISCUSSION THREAD

**Discussion by yylmxyzc** [the1812/Bilibili-Evolved#3756] [提问 / Q&A] [answered]
这是正常设计还是bug?如果是正常设计的话有点不方便，每次回到首页都要在其他地方点一下才会收回。

**Comment by the1812** [comment, marked as answer]
正常设计，避免在里面打字的时候收起

**Reply by yylmxyzc** [reply]
动态预览没有需要打字的地方吧？而且进入动态，首页的动态预览也不会收起，这有点不习惯。

**Reply by the1812** [reply]
应该可以再优化下

**Reply by yylmxyzc** [reply]
好吧，收藏和历史搜索需要打字，之前有打字时会收起的bug。但是每次进入动态后在主页还要点击下其他地方弹窗才会消失确实麻烦。
